lib/uksched/sched.c

```diff
@@ -107,6 +107,8 @@ int uk_sched_set_default(struct uk_sched *s)
 			prev->next = this->next;
 			this->next = head->next;
 			head = this;
+			//update the default scheduler to head
+			uk_sched_head = head;
```

Comment:
```diff
- 		//update the default scheduler to head
+ 		// Update the default scheduler to head
```

Reply:
Thank you guiding me. I made all the requested change to the comment and sign-off the commit.